main.go

```diff
@@ -96,6 +98,19 @@ func main() {
 		fmt.Fprintf(os.Stderr, "invalid option:%q. The -format flag must be one of: basic, plain or markdown", *formatPtr)
 		return
 	}
+	var sorter parse.PackageSorter
+	switch *sortPtr {
+	case "name":
+		sorter = parse.SortByPackageName
+	case "elapsed":
+		sorter = parse.SortByElapsed
+	case "cover":
+		sorter = parse.SortByCoverage
+	default:
+		fmt.Fprintf(os.Stderr, "invalid option:%q. The -format flag must be one of: name, elapsed or cover\n", *sortPtr)
```

Comment:
Wrong flag name in error string: `-format` .. should be whatever we decided in the comment above.